KsApi/fragments/UserFragment.graphql

```diff
@@ -0,0 +1,7 @@
+fragment UserFragment on User {
```

Comment:
Maybe we can make a small image size fragment called `UserCommentFragment` with `200` as the image width. Just because it'll be a smaller download size for our comments cells to use,  maybe even in `Backing`,  it could in theory help performance for repeatedly used ui components like table view cells.

Reply:
I'll look at doing this in a subsequent PR if it makes sense 👍